### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by sercansimsek**
- [DEMO LINK](https://sercansimsek.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  width: $widthCalendar;
+
+  @each $number, $day in $weekDays {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($number - 1) * ($sizeDay + $gapDay);
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
```
Comment (SalehAkaJim) [inline src/styles/blocks/calendar.scss:14]: You can add this to html

```
html {
  font-family: Arial, Helvetica, sans-serif;
  font-size: 30px;
  box-sizing: border-box;
}
```

File: src/styles/blocks/main.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+}
```
Comment (SalehAkaJim) [inline src/styles/blocks/main.scss:7]:
```suggestion
  height: 100vh;
}
```

**Review by SalehAkaJim — CHANGES_REQUESTED** [review]

File: src/styles/blocks/calendar.scss (hunk 1 above)
Comment (sercansimsek) [reply src/styles/blocks/calendar.scss:14]: I tried it once more and it did work at all. It seems like it is the best solution for me to keeping it there :D

**Review by SalehAkaJim — APPROVED** [review]

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  flex-wrap: wrap;
+  gap: $gapDay;
+  width: $widthCalendar;
+
+  @each $number, $day in $weekDays {
+    &--start-day--#{$day} :first-child {
+      margin-left: ($number - 1) * ($sizeDay + $gapDay);
+    }
+  }
+
```
Comment (meforoma) [inline src/styles/blocks/calendar.scss:12]: hmm, i can't have a 31-day month? i should be able to.
neither can i show 28 days
`class="calendar calendar--start-day--tue calendar--month-length--28"`
you need another `@for` loop for this 

consult with Ashkar brothers if you don't know how to approach it, k?

**Review by meforoma — CHANGES_REQUESTED** [review]
see just the one, but important comment

**Pull request closed without merging**